### ISSUE TITLE
Cigarettes per day

### ISSUE THREAD

**Issue by phillip-toone** [SaltLakeTechnicalCenter/COHb#11]
Allow the smoker status to be obtained from a known number of cigarettes smoked per day.

`Measurement Entry #12`

**Comment by phillip-toone** [comment]
The way this legacy program deals with smoking is unnecessarily complex.  It relates the initial concentration of COHb in the blood and the steady state exposure to CO in a particular phase to "smoker status" by interpolating values between defined values.  But then goes on to define "smoker status" by the number of cigarettes smoked and uses the following curve fit between "smoker status" and number of cigarettes smoked.
```
Levl = INT(-2.7895 + SQR(7.7813 + 14.1052 * Cigs) / .70526 + .5) / 10 - .06
where Levl is "smoker status" and Cigs is the number of cigarettes smoked.
```
It seems to me that the use of "smoker status" could and should be elminated and that the desired parameters (initial concentration of COHb in the blood and the steady state exposure to CO in a particular phase) should be related directly to the number of cigarettes smokes via interpolation as shown below.

![smokerStatus](https://user-images.githubusercontent.com/5975365/87109178-30673180-c221-11ea-9d6e-2a52951e974d.png)

**Comment by phillip-toone** [comment]
I still need to add the load/save features for this fix.

**Comment by phillip-toone** [comment]
Issue #11 has been opened and closed over a number of commits where I struggled with the best way to deal with the  initial concentration of COHb in the blood.  This issue is finally resolved and the user can have this parameter determine by either 1) the number of cigarettes smoked per day or 2) the smoker status.  At first glance this seems to be the way the legacy program worked but it determined the smoker status from the number of cigarettes smoked per day.  This involved unnecessary intermediate step that have been eliminated.  In an attempt to simply the user input the default is set for the employee as a non smoker where neither of these two options are available.

In summary the program doesn't really do anything differently that it did before but this has helped me lay the groundwork for how I am going to deal with the issue of Issue #7, smoking on the job.
